MAVProxy/mavproxy.py

```diff
@@ -269,15 +269,18 @@ def cmd_watch(args):
 
 def generate_kwargs(args):
     kwargs = {}
-    module_components = args.split(":{", 1)
+    module_components = args.split(":", 1)
```

Comment:
When there is a `:'{` the old code breaks

Reply:
I have not found this to be the case. I believe the problem is with `modlist = module.split(',')` cutting the json string into what it thinks are module names

Reply:
Yes, `module.split(',')` cutting the json string into what it thinks are module names is the problem.